org.keshavnrj.wonderwall.yaml

```diff
@@ -0,0 +1,46 @@
+app-id: org.keshavnrj.wonderwall
+base: io.qt.qtwebkit.BaseApp
+base-version: '5.11'
+runtime: org.kde.Platform
+runtime-version: '5.11'
+sdk: org.kde.Sdk
+command: wonderwall
+finish-args:
+  - --share=network
```

Comment:
are network permissions necessary?

Reply:
Yes, the pulls wallpapers from web. That's the only reason my application require network.